docs/Agent/user-guide/man_page.md

```diff
@@ -333,107 +265,205 @@ pbench-register-tool-trigger, Registers the tool trigger
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+Path to the pbench-agent configuration file.
 
--g, --group,--groups STR  
+`-g`, `--group`, `--groups` STR  
 Registers the trigger in the <group-name> group. If no group is specified, the 'default'
 
---start-trigger STR  
+`--start-trigger` STR  
 [required]
 
---stop-trigger STR  
+`--stop-trigger` STR  
 [required]
 
---help  
+`--help`  
 Show this message and exit.
 
+
 ---
 
-### pbench-results-move
+##### pbench-user-benchmark
 
 ---
 
 **NAME**  
-pbench-results-move, Move results tarball to the server
+`pbench-user-benchmark`, Control start/stop/post-process-tools.
 
 **SYNOPSIS**  
-`pbench-results-move[-C][--controller][--token][--delete][--xz-single-threaded][--show-server][--help]
-`
+`pbench-user-benchmark[-C][--tool-group][--iteration-list][--sysinfo][--pbench-pre][--pbench-post][--use-tool-triggers][--no-stderr-capture] --<command to run>`
 
 **DESCRIPTION**  
-Move result directories to the configured Pbench server.This command is used when pushing result tarball using token.
+
+Collects data from the registered tools while running a user-specified action. This can be a specific synthetic benchmark workload, a real workload, or simply a delay to measure system activity.
+
+Here are the steps involved
+
+- Invoking pbench-user-benchmark with your workload generator as an argument: that will start the collection tools on all the hosts
+- Next, it will run your workload generator; when that finishes, it will stop the collection tools on all the hosts
+- Finally, the postprocessing phase will gather the data from all the remote hosts and run the postprocessing tools on everything.
 
 **OPTIONS**  
- -C, --config PATH  
-Path to a pbench-agent configuration file (defaults to the '\_PBENCH_AGENT_CONFIG' environment variable,if defined) [required]
+`-C`, `--config` PATH  
+This option is required if not provided by the `_PBENCH_AGENT_CONFIG` environment variable.
+Path to the pbench-agent configuration file.
 
---controller STR  
-Override the default controller name
+`--tool-group`  
+The tool group to use for the list of tools
 
---token STR  
-pbench server authentication token [required]
+`--iteration-list`  
+A file containing a list of iterations to run for the provided script;
+the file should contain one iteration per line . With a leading '#' (hash) character used for comments and blank lines are ignored.
+Each iteration line should use alpha-numeric characters before the first space to name the iteration, with the rest of the line provided as arguments to the script;  
+_NOTE: --iteration-list is not compatible with --use-tool-triggers_
 
---delete / --no-delete  
-Remove local data after successful copy [default: delete]
+`--sysinfo` STR,[STR]  
+comma-separated values of system information to be collected; available: default, none, all, ara, block, insights, kernel_config, libvirt, security_mitigations, sos, stockpile, topology
```

Comment:
Since the comma is only required if there is a subsequent value, it should be inside the square brackets:
```
`--sysinfo` STR[,STR...]
```
and the elipsis indicates that there can be more than a second string.